packages/app/src/cli/services/deploy/upload.ts

```diff
@@ -181,7 +197,14 @@ export async function uploadExtensionsBundle(
     if (result.appDeploy.appVersion) {
       deployError = result.appDeploy.userErrors.map((error) => error.message).join(', ')
     } else {
```

Comment:
Is the `invalid` something that we control though? Or is that common for all deploy validation checks?

Reply:
IMO we should be consistent with the other extension deploy validation errors and reuse the same experience. This would also allow partners to see the full list of deploy validation errors, not just a single unsupported version one.

In that case, we have control over the message shown to partners (see `message` in the above JSON object).

@MeredithCastile WDYT?